x-pack/platform/plugins/shared/streams_app/public/components/stream_data_quality/index.tsx

```diff
@@ -22,6 +23,15 @@ export function StreamDetailDataQuality({
 
   const controller = useDatasetQualityController(definition, true, refreshDefinition);
 
+  const { onPageReady } = usePerformanceContext();
+
+  // Telemetry for TTFMP (time to first meaningful paint)
+  useEffect(() => {
+    if (definition && controller) {
+      onPageReady();
```

Comment:
Is this the right place to call it? For example, in the main page of Data Set quality, it's used from the [DatasetsQualityIndicators component](https://github.com/elastic/kibana/blob/c2136ee32f828d9c57d03603fa00aa4b3efb469f/x-pack/platform/plugins/shared/dataset_quality/public/components/dataset_quality/summary_panel/datasets_quality_indicators.tsx#L39).